src/Tools/Microsoft.dotnet-openapi/src/Commands/BaseCommand.cs

```diff
@@ -272,6 +275,56 @@ internal async Task<string> DownloadGivenOption(string url, CommandOption fileOp
             }
         }
 
+        /// <summary>
+        /// Retries every 1 sec for 60 times by default.
```

Comment:
No backoff? Suggest starting w/ waiting a half second and doubling that each iteration until about a minute has passed. Should probably track total time because `HttpClient.Timeout`.

Reply:
Same as above.